mne/preprocessing/tests/test_stim.py

```diff
@@ -64,13 +66,36 @@ def test_stim_elim():
     test_tminidx = int(0.015 * raw.info['sfreq'])
     test_tmaxidx = int(0.035 * raw.info['sfreq'])
 
-    raw = eliminate_stim_artifact(raw, events, event_id=1, tmin=tmin,
+    raw = fix_stim_artifact_raw(raw, events, event_id=1, tmin=tmin,
                                   tmax=tmax, mode='linear')
     data, times = raw[:, (tidx + test_tminidx):(tidx + test_tmaxidx)]
     diff_data0 = np.diff(data[0])
     diff_data0 -= np.mean(diff_data0)
     assert_array_almost_equal(diff_data0, np.zeros(len(diff_data0)))
-    raw = eliminate_stim_artifact(raw, events, event_id=1, tmin=tmin,
+    raw = fix_stim_artifact_raw(raw, events, event_id=1, tmin=tmin,
                                   tmax=tmax, mode='window')
     data, times = raw[:, (tidx + test_tminidx):(tidx + test_tmaxidx)]
     assert_true(np.all(data) == 0.)
+
+
+def test_stim_fix_evoked():
+    """Test eliminate stim artifact"""
+    raw = Raw(raw_fname, preload=True)
+    events = read_events(event_fname)
+    event_id = 1
+    tmin = -0.02
+    tmax = 0.02
```

Comment:
nitpick, but the last 3 lines can be combined into one by `tmin, tmax, event_id = -0.02, 0.02, 1`